.gitignore

```diff
@@ -59,7 +59,7 @@ node_modules
 .directory
 
 # Ignore plugins
-flaskbb/plugins/*
+# flaskbb/plugins/*
```

Comment:
Do a force add instead of removing this.

Reply:
thanks. I am not very familiar with git commands. I will try that later.

Reply:
The command you'll want is `git add - f`, be sure to target the exact file though as the f flag tells git "I'm serious, actually add this even though we said to ignore it"

Reply:
thanks for teaching :)

Reply:
@kwijybo We're changing how plugins are done. The plugins module is becoming a collection of tools to manage plugins. In the future plugins will be installed via pip and loaded via entrypoints